DigitalLearningSolutions.Data/Services/NotificationService.cs

```diff
@@ -53,8 +53,8 @@ public async Task SendUnlockRequest(int progressId)
 
             var baseUrlConfigOption = refactoredTrackingSystemEnabled
                 ? configuration.GetAppRootPath()
-                : configService.GetConfigValue(ConfigHelper.CurrentSystemBaseUrlName);
-            if (baseUrlConfigOption == null)
+                : configuration.GetCurrentSystemBaseUrl();
+            if (string.IsNullOrEmpty(baseUrlConfigOption))

```

Comment:
I can't mark it against the actual lines where the change is needed, but by changing this, you're breaking the link in the emails again. This will currently produce https://oldsitebaseurl.co.uk/CourseDelegates rather than the https://oldsitebaseurl.co.uk/Tracking/CourseDelegates that is the correct path.

The database TrackingSystemBaseURL value that was used in the old version of this code is https://oldsitebaseurl.co.uk/tracking/
vs the appsettings value of https://oldsitebaseurl.co.uk/